core/src/main/java/org/mobilitydata/gtfsvalidator/input/GtfsZipFileInput.java

```diff
@@ -46,32 +47,24 @@ public GtfsZipFileInput(ZipFile zipFile, String zipFileName) {
       ZipArchiveEntry entry = entries.nextElement();
       String entryName = entry.getName();
       // check if the first entry is a directory with the name of the zip file
-      if (entry.isDirectory() && zipFileName.equals(entryName.replaceFirst("/", ""))) {
+      String firstEntryName = entryName.replaceFirst("/", "");
+      if (entry.isDirectory() && zipFileName.equals(firstEntryName)) {
         isMacZip = true;
```

Comment:
Now that the two usages of isMacZip is confined to the constructor, it seems to it could be a local variable. No?